api/src/common/interface/user.interface.ts

```diff
@@ -0,0 +1,5 @@
+export interface User {
+  sub: string;
+  given_name: string;
+  family_name?: string;
+}
```

Comment:
no location?

Reply:
this is the user type from token.

For database user, we have userEntity.

But I think I may miss something, let me double check.